grails-app/controllers/org/broadinstitute/orsp/api/DataUseController.groovy

```diff
@@ -23,4 +43,56 @@ class DataUseController extends AuthenticatedController {
         render queryService.findDataUseRestrictions(pagination) as JSON
     }
 
+    def findRestriction() {
+        UtilityClass.registerSampleCollectionMarshaller()
+        DataUseRestriction restriction = DataUseRestriction.findById(params.id)
+        List<String> summary = consentService.getSummary(restriction)
+        Issue consent = queryService.findByKey(restriction.consentGroupKey)
+        String consentGroupName = consent.summary
+        ConsentResource consentResource = consentService.buildConsentResource(restriction, consentGroupName)
+        Collection<ConsentCollectionLink> collectionLinks = queryService.findCollectionLinksByConsentKey(restriction.consentGroupKey)
+        collectionLinks.retainAll { it.sampleCollectionId != null }
+        Set<SampleCollection> samples = collectionLinks.collect{ it.sampleCollection }
+        JSON.use(UtilityClass.SAMPLES) {
+            render([restriction    : restriction,
+                    restrictionUrl : restriction.getConsentUrl(getConsentServiceUrl()),
+                    summary        : summary,
+                    consent        : consent,
+                    samples: samples,
```

Comment:
nit: alignment - if you want to align the other arguments, this one should be too.